rotmat.py

```diff
@@ -401,3 +416,59 @@ def plane_intersection(self, plane, forward_only=False):
             return None
         return (self.vector * d) + self.point
 
+class Rotation(object):
+    def __init__(self, name, roll, pitch, yaw):
+        self.name = name
+        self.roll = roll
+        self.pitch = pitch
+        self.yaw = yaw
+        self.r = Matrix3()
+        self.r.from_euler(self.roll, self.pitch, self.yaw)
+        self.rt = self.r.transposed()
+
+# the rotations used in APM
+rotations = [
+    Rotation("ROTATION_NONE",                      0,   0,   0),
```

Comment:
why are we maintaining our Rotation instead of using MAVLINK one's ? the diff is quite small

Reply:
we could switch to that, good future enhancement, though it would need enum description parsing